takwimu/templates/takwimu/about/support-services.html

```diff
@@ -0,0 +1,21 @@
+{% extends 'takwimu/_layouts/page.html' %}
+
+{% block content %}
+
+<div class="py-5 container border-bottom" style="max-width: 800px;">
+  <h1 class="text-center font-weight-bold display-3 mb-5">Support Services</h1>
+  <p>Takwimu was launched in 2018 in response to the problem posed by poor access to quality data, which limits the effectiveness of development policy and programming in Africa.</p>
+  <p>Conceived by a consortium of partners - africapractice, Code 4 Africa and iHub - and funded by the Bill and Melinda Gates Foundation, Takwimu is designed to provide a free and open data service which empowers African development actors with the insights to achieve greater impact.</p>
+  <p>The website offers concise, objective, data-rich profiles on ten African countries (Burkina Faso, Democratic Republic of Congo, Ethiopia, Kenya, Nigeria, Senegal, South Africa, Tanzania, Uganda and Zambia) in English and French. Work is underway to expand coverage to more countries and to make the content available in additional languages.</p>
+</div>
+
+{% include 'takwimu/_includes/sections/services.html' %}
+
+{% include 'takwimu/_includes/sections/enquire.html' %}
+
+
+{% include 'takwimu/_includes/partners.html' %}
+
+{% include 'takwimu/_includes/sections/updates.html' %}
+
+{% endblock %}
```

Comment:
Is this used anymore? I thought we are using a different template for the support services page?